service/create/encrypt.go

```diff
@@ -36,18 +39,21 @@ type encryptedTLSAssets TLSassets
 func createRawTLSAssets(assets certificatetpr.AssetsBundle) *rawTLSAssets {
 	// TODO refactor this with a for loop iterating over components and asset types
```

Comment:
I want to get apiserver unblocked but it would be good to tackle this TODO. Shall I do this once we've got apiserver working?